src/app/frontend/replicationcontrollerdetail/replicationcontrollerdetail.scss

```diff
@@ -85,6 +85,15 @@ $table-cell-height-half: $table-cell-height / 2;
     white-space: normal;
     word-break: break-all;
   }
+
+  &.kd-replicationcontrollerdetail-metric-table-cell {
+    text-align: right;
+    white-space: nowrap;
+  }
+
+  &.kd-replicationcontrollerdetail-sparkline-table-cell {
+    padding-left: 1.25 * $baseline-grid;
```

Comment:
Can we avoid 1.25? We tend to have $baseline-grid everywhere, unless justified otherwise.